Helpers.gs

```diff
@@ -778,33 +776,37 @@ function parseNotificationTime(notificationString){
 /**
 * Sends an email summary with added/modified/deleted events.
 */            
-function sendSummary() {
-  var subject;
-  var body;
+function sendSummary(email, addedEvents, modifiedEvents, removedEvents) {
+  if (email == "") {
+    Logger.log("A summary email will not be sent because an email address has not been provided.")
```

Comment:
I would like to avoid adding extra log lines as a line like this could seem like an error to a user. It would make sense to have this if you kept the `emailSummary` bool (since the user *requested* an email, but didn't provide an email address), but if you're removing that bool, let's also remove this log line

Reply:
Ok, fair enough. I prefer verbosity so people never have to guess why things did or did not happen. Since it's a log file, it makes sense to me to emit messages when decisions are made.

Reply:
The new GAS UI also shows log messages at the bottom when running (rather than a separate window) so users are paying attention to them more. We've gotten more comments about the "Timezone _____ unsupported! Using ____ instead" and whether that was an error, something will mess up their calendar, etc